share/doc/wake/logging.md

```diff
@@ -20,22 +20,22 @@ and a display format (color and intensity).
 ## Using & Creating Loggers
 
 `wake` has a number of built in loggers in which all start with `log...`.
-You can also make your own logger using `mkLogLevel`.
+You can also make your own `LogLevel` tuple.
 You can use this to redirect wake output to a dedicated stream which can
 later be filtered on.
 
-| wake `def`        | Color/Intensity | log level     |
-|-------------------|-----------------|---------------|
-| logDebug          |     Blue        | "debug"       |
-| logInfo           |     Gray        | "info"        |
-| logEcho           |     Default     | "echo"        |
-| logInteractive    |     Cyan        | "interactive" |
-| logReport         |     Magenta     | "report"      |
-| logWarning        |     Yellow      | "warning"     |
-| logError          |     Red         | "error"       |
-| logNever          |     Default     | "null"        |
-| LogLevel "foo"    |     Default     | "foo"         |
-| LogLevel "bar"    |     Default     | "bar"         |
+| wake `def`        |  log level    |
+|-------------------|---------------|
+| logDebug          | "debug"       |
+| logInfo           | "info"        |
+| logEcho           | "echo"        |
+| logInteractive    | "interactive" |
+| logReport         | "report"      |
+| logWarning        | "warning"     |
+| logError          |  "error"      |
```

Comment:
nit: can you realign the  left quote on `"error"` and `"null"`

Reply:
oops just saw this